README.md

```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
+
+## Building a Spectrum OS Image
+
+Clone sources of spec and nix-spec (cross-compile branches for i.MX 8QXP):
+
+	$ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/nixpkgs-spectrum/
+    $ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/spectrum/
+
+Set up the Spectrum binary cache.
+
+* Change non-NixOS machine configuration.
+
+	Note the following:
+
+	- The custom binary cache is located here: <http://binarycache.vedenemo.dev>.
+	- The public key of this cache is:
+      `binarycache.vedenemo.dev:Yclq5TKpx2vK7WVugbdP0jpln0/dPHrbUYfsH3UXIps=`
+    - Spectrum OS own binary cache is located here: <https://cache.dataaturservice.se/spectrum/>.
+    - The public key of this cache is:
+        `spectrum-os.org-1:rnnSumz3+Dbs5uewPlwZSTP0k3g/5SRG4hD7Wbr9YuQ=`
+
+    All configurations are made in the `/etc/nix/nix.conf` file. It should be possible to carry everything to the NixOS configuration with little effort.
```

Comment:
Proposal - let's simplify this chapter by listing only:
1. vedenemo binary-cache details and link to Spectrum binary cache https://spectrum-os.org/doc/binary-cache.html
2. Link to Binary Cache usage - https://nixos.wiki/wiki/Binary_Cache#Using_a_binary_cache 

More detailed comments why this is chapter as of now is problematic:

- it only covers non-NixOS based setup and does not cover NixOS based setups
- it replicates info already available from Spectrum and NixOS - it's easier for us to keep info up to date by not replicating

Reply:
Made some attempts. If something comes up, lets' discuss this.